services/graph/pkg/config/unified_roles.go

```diff
@@ -0,0 +1,6 @@
+package config
+
+// UnifiedRoles contains all settings related to unified roles.
+type UnifiedRoles struct {
+	AvailableRoles []string `yaml:"available_roles" env:"UNIFIED_ROLES_AVAILABLE_ROLES" desc:"A list of roles that are available for assignment." introductionVersion:"%%NEXT%%"`
```

Comment:
Shouldn't this be something like `OCIS_AVAILABLE_ROLES`? Usually ocis envvars start with the service name or `OCIS`